src/main/java/de/ellpeck/actuallyadditions/mod/util/StackUtil.java

```diff
@@ -36,9 +36,14 @@ public static ItemStack validateCheck(ItemStack stack){
     }
 
     public static boolean isValid(ItemStack stack){//Stacks are nonnull.  If we are making null stacks we're stupid anyway.
-    	if(stack == null) AwfulUtil.callTheFuckinPolice("Oh yeah some idiot somewhere threw a null itemstack at us, might've been us, but whatever");
+    	if(stack == null){
+    	    AwfulUtil.callTheFuckinPolice("Oh yeah some idiot somewhere threw a null itemstack at us, might've been us, but whatever");
+    	    return false;
+        }
     	Item i = stack.getItem();
-    	if(i instanceof IDisableableItem) return !((IDisableableItem) i).isDisabled();
+    	if(i instanceof IDisableableItem){
+    	    return !((IDisableableItem) i).isDisabled();
+        }
         return !stack.isEmpty();
```

Comment:
Hey look its you formatting my shit again for no reason...
You don't even touch this class.

Reply:
This place - isn't my fault; there was mixed indenting (tabs and spaces). Replace to spaces?